actions/drain_node_handler.go

```diff
@@ -277,7 +274,16 @@ func (h *drainNodeHandler) evictPod(ctx context.Context, pod v1.Pod) error {
 	return nil
 }
 
-func daemonSetPod(pod v1.Pod) bool {
-	controller := metav1.GetControllerOf(&pod)
-	return controller != nil && controller.Kind == "DaemonSet"
+func isDaemonSetPod(p *v1.Pod) bool {
+	return isControllerOf(p, "DaemonSet")
+}
+
+func isStaticPod(p *v1.Pod) bool {
+	return isControllerOf(p, "Node")
+}
+
+func isControllerOf(p *v1.Pod, kind string) bool {
```

Comment:
random nitpic: feels like this should be `isControlledBy` ?

Reply:
Fixed, isControlledBy is correct method here, thanks.